backend/api/Database/Models/Inspection.cs

```diff
@@ -50,16 +53,24 @@ public Inspection(Inspection copy)
         // ReSharper disable once AutoPropertyCanBeMadeGetOnly.Local
         public string? IsarStepId { get; private set; } = Guid.NewGuid().ToString();
 
+        public Position InspectionTarget { get; set; }
+
         [Required]
         public InspectionStatus Status
         {
             get => _status;
             set
             {
                 _status = value;
-                if (IsCompleted && EndTime is null) { EndTime = DateTime.UtcNow; }
-
-                if (_status is InspectionStatus.InProgress && StartTime is null) { StartTime = DateTime.UtcNow; }
+                if (IsCompleted && EndTime is null)
```

Comment:
Why does it format into one line in other files and here it split into multiple?